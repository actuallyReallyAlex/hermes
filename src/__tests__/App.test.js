describe('Fake Test', () => {
  it('Should pass.', () => {
    expect(1 + 1).toBe(2)
  })
})
