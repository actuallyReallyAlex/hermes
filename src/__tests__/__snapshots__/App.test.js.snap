// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should render the <App /> component. 1`] = `
<DocumentFragment>
  <div
    class="StyledBox-sc-13pk1d4-0 lfMSfr"
    id="outer-container"
    style="overflow: hidden;"
  >
    <div>
      <div
        class="bm-menu-wrap"
        id=""
        style="position: fixed; z-index: 1100; width: 300px; height: 100%; transition: all 0.5s;"
      >
        <div
          class="bm-menu"
          style="height: 100%; box-sizing: border-box; overflow: auto;"
        >
          <nav
            class="bm-item-list"
            style="height: 100%;"
          >
            <div
              class="bm-item StyledBox-sc-13pk1d4-0 fWrrBh"
              style="display: block;"
              tabindex="0"
            >
              <h1
                class="StyledHeading-sc-1rdh4aw-0 rfVJR"
              >
                Hermes
              </h1>
              <h2
                class="StyledHeading-sc-1rdh4aw-0 yhirM"
              >
                View
              </h2>
              <div
                class="StyledBox-sc-13pk1d4-0 fWrrBh"
              >
                <a
                  class="StyledAnchor-sc-1rp7lwl-0 fwWNdB"
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 hfsUvK"
                  >
                    <span
                      class="StyledText-sc-1sadyjn-0 fWSbXS"
                    >
                      Market
                    </span>
                  </div>
                </a>
                <div
                  class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hPsznL"
                />
                <a
                  class="StyledAnchor-sc-1rp7lwl-0 fwWNdB"
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 hfsUvK"
                  >
                    <span
                      class="StyledText-sc-1sadyjn-0 fWSbXS"
                    >
                      Planets
                    </span>
                  </div>
                </a>
                <div
                  class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hPsznL"
                />
                <a
                  class="StyledAnchor-sc-1rp7lwl-0 fFtORL"
                  disabled=""
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 hfsUvK"
                  >
                    <span
                      class="StyledText-sc-1sadyjn-0 fWSbXS"
                    >
                      Ship
                    </span>
                  </div>
                </a>
              </div>
              <div
                class="StyledBox-sc-13pk1d4-0 lfluzk"
              >
                <h2
                  class="StyledHeading-sc-1rdh4aw-0 gBloPj"
                >
                  Cash:
                </h2>
                <span
                  class="StyledText-sc-1sadyjn-0 fWSbXS"
                >
                  100
                </span>
              </div>
              <div
                class="StyledBox-sc-13pk1d4-0 egwKgD"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 fWrrBh"
                >
                  <h3
                    class="StyledHeading-sc-1rdh4aw-0 ftOecI"
                  >
                    Contracts
                  </h3>
                  <div
                    class="StyledBox-sc-13pk1d4-0 jhciyq"
                  >
                    <button
                      class="StyledButton-sc-323bzc-0 lnKzGb"
                      type="button"
                    >
                      Place Buy Order
                    </button>
                    <button
                      class="StyledButton-sc-323bzc-0 lnKzGb"
                      type="button"
                    >
                      View Contracts
                    </button>
                  </div>
                </div>
              </div>
              <h3
                class="StyledHeading-sc-1rdh4aw-0 cVyJbT"
              >
                Settings
              </h3>
              <div
                class="StyledBox-sc-13pk1d4-0 ekjAwv"
              >
                <button
                  class="StyledButton-sc-323bzc-0 jfbKsv"
                  data-testid="button-export"
                  type="button"
                >
                  Export Game
                </button>
                <div
                  class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hyyRSo"
                />
                <div
                  class="StyledBox-sc-13pk1d4-0 fWrrBh"
                >
                  <button
                    class="StyledButton-sc-323bzc-0 jfbKsv"
                    type="button"
                  >
                    Import Game
                  </button>
                  <input
                    accept=".json"
                    data-testid="input-import"
                    style="display: none;"
                    type="file"
                  />
                </div>
              </div>
            </div>
          </nav>
        </div>
        <div>
          <div
            class="bm-cross-button"
            style="position: absolute; width: 24px; height: 24px; right: 8px; top: 8px;"
          >
            <span
              style="position: absolute; top: 6px; right: 14px;"
            >
              <span
                class="bm-cross"
                style="position: absolute; width: 3px; height: 14px; transform: rotate(45deg);"
              />
              <span
                class="bm-cross"
                style="position: absolute; width: 3px; height: 14px; transform: rotate(-45deg);"
              />
            </span>
            <button
              style="position: absolute; left: 0px; top: 0px; width: 100%; height: 100%; margin: 0px; padding: 0px; font-size: 0; background: transparent; cursor: pointer;"
              tabindex="0"
            >
              Close Menu
            </button>
          </div>
        </div>
      </div>
      <div>
        <div
          class="bm-burger-button"
          style="z-index: 1000;"
        >
          <span>
            <span
              class="bm-burger-bars"
              style="position: absolute; height: 20%; left: 0px; right: 0px; top: 0%; opacity: 1;"
            />
            <span
              class="bm-burger-bars"
              style="position: absolute; height: 20%; left: 0px; right: 0px; top: 40%; opacity: 1;"
            />
            <span
              class="bm-burger-bars"
              style="position: absolute; height: 20%; left: 0px; right: 0px; top: 80%; opacity: 1;"
            />
          </span>
          <button
            style="position: absolute; left: 0px; top: 0px; width: 100%; height: 100%; margin: 0px; padding: 0px; font-size: 0; background: transparent; cursor: pointer;"
          >
            Open Menu
          </button>
        </div>
      </div>
    </div>
    <div
      class="StyledBox-sc-13pk1d4-0 jWgukJ"
      id="page-wrap"
      style="max-width: calc(100% - 300px); transform: translate3d(300px, 0, 0); transition: all 0.5s;"
    >
      <div>
        <h2>
          Your Ship
        </h2>
        <h3>
          Cargo
        </h3>
        <div
          class="StyledBox-sc-13pk1d4-0 fWrrBh"
        >
          <span
            class="StyledText-sc-1sadyjn-0 bnqisW"
          >
            Volume Remaining:
          </span>
          <span
            class="StyledText-sc-1sadyjn-0 fWSbXS"
          >
            5
          </span>
        </div>
        <div
          class="Table_tableWrapper__1fXsJ Table_tableScroll__zRwCt undefined"
        >
          <table
            class="Table_table__1DoSj"
          >
            <thead>
              <tr>
                <th>
                  Name
                </th>
                <th>
                  Destination
                </th>
                <th>
                  Quantity
                </th>
                <th>
                  Remove
                </th>
              </tr>
            </thead>
            <tbody />
          </table>
        </div>
        <h3>
          Location:
        </h3>
        <div
          class="StyledBox-sc-13pk1d4-0 jTvEOL"
        >
          <span
            class="StyledText-sc-1sadyjn-0 isVaoK"
          >
            Value:
          </span>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hPsznL"
          />
          <span
            class="StyledText-sc-1sadyjn-0 gNZQxf"
          />
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hPsznL"
          />
          <span
            class="StyledText-sc-1sadyjn-0 isVaoK"
          >
            Name:
          </span>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hPsznL"
          />
          <span
            class="StyledText-sc-1sadyjn-0 gNZQxf"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
